sharded/shardIDProvider.go

```diff
@@ -5,6 +5,7 @@ import (
 	"math"
 )
 
+const minNumOfShards = 1
```

Comment:
does it make sense to have 1 shard in shard ID provider? :D 
I suggest to change this to 2

Reply:
changed to 2 :+1: